plugins/sigma/integration/data/zeek

```diff
@@ -0,0 +1 @@
+../../../../vast/integration/data/zeek
```

Comment:
This file needs to be resolved manually in the Dockerfile.

Reply:
Look for ln in the Dockerfile, then add this symlimk there as well. Docker does not copy symlimks.

Reply:
Tried this in 3719181.